examples/04_glm_first_level/plot_design_matrix.py

```diff
@@ -34,7 +34,7 @@
 onsets = [30.0, 70.0, 100.0, 10.0, 30.0, 90.0, 30.0, 40.0, 60.0]
 # Next, we simulate 6 motion parameters jointly observed with fMRI acquisitions
 rng = np.random.default_rng(42)
-motion = np.cumsum(rng.randn(n_scans, 6), 0)
+motion = np.cumsum(rng.standard_normal((n_scans, 6)), 0)
```

Comment:
need to investigate but this was crashing for me without this fix

Reply:
Actually this also crashes the doc build on main: https://github.com/nilearn/nilearn/actions/runs/11343634049/job/31548674527#step:14:15870

So this fix is needed.